test/integration/policy_compliance_operator_cis_scan_test.md

```diff
@@ -0,0 +1 @@
+The test for `policy-compliance-operator-cis-scan.yaml` can be found in the [Compliance Operator](./policy_comp_operator_test.go) test.
```

Comment:
Are these README files still needed? Not a big deal to me. Just checking.

Reply:
I guess I'm on the fence, too. I do think they're helpful for transparency/traceability if someone is wondering whether the policies are tested since they don't fit the patterns of the other policies that have a dedicated test file, but they are extra files in a large list...

Reply:
I think we could either remove these files or just provide one single README to explain what each test case does.